### PR TITLE
[WIP] APPS-138: Added the console to generate Payment module

### PR DESCRIPTION
**Pull request by matweew**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "spryker-sdk/spryk-payment",
+  "type": "library",
+  "description": "SprykPayment module",
+  "license": "proprietary",
+  "require": {
+    "php": ">=7.3",
+    "spryker/symfony": "^3.2.2"
+  },
```
[the remark this answers is not in the document]
Comment (vol4onok) [reply composer.json:9]: I added it to dependency.json

File: src/SprykerSdk/SprykPayment/SprykPaymentFacade.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    /**
+     * @codeCoverageIgnore
+     *
+     * @return \SprykerSdk\SprykPayment\SprykPaymentFactory
+     */
+    protected function getFactory(): SprykPaymentFactory
+    {
+        if ($this->factory === null) {
+            $this->factory = new SprykPaymentFactory();
```
Comment (olehnovatskyi-spryker) [inline src/SprykerSdk/SprykPayment/SprykPaymentFacade.php:46]: Why is `SprykPaymentFactory` created here?

File: composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (matweew) [reply composer.json:9]: `dependency.json` is only for our Spryker tooling (depencecy checker) composer doesn't know about it.
But you call spryk commands directly in your module

File: src/SprykerSdk/SprykPayment/SprykPaymentFacadeInterface.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\SprykPayment;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+interface SprykPaymentFacadeInterface
+{
+    /**
```
[the remark this answers is not in the document]
Comment (vol4onok) [reply src/SprykerSdk/SprykPayment/SprykPaymentFacadeInterface.php:14]: it's not API method, we don't have a mechanism of facade and etc.

**Comment by dereuromark** [comment]
Is this still happening? Or should we remove this repo now?

**Comment by pushokwhite** [comment]
> Is this still happening? Or should we remove this repo now?

It's abandoned, please remove.

**Pull request closed without merging**